courses/views_test.py

```diff
@@ -30,7 +33,8 @@ def test_lists_live_programs(self):
         resp = self.client.get(reverse('program-list'))
 
         assert len(resp.json()) == 1
-        assert prog.title == resp.json()[0]['title']
+        expected = ProgramSerializer(context={"request": Mock(user=self.user)}).to_representation(prog)
```

Comment:
We should avoid calling `.to_representation()` on a Serializer. The standard API is to pass the instance to the constructor, and access the `data` property. Like this:
```python
context = {"request": Mock(user=self.user)}
data = ProgramSerializer(prog, context=context).data
assert [data] == resp.json()
```